library/cpp/include/wavemap/core/indexing/ndtree_index.h

```diff
@@ -21,7 +21,12 @@ struct NdtreeIndex {
   static constexpr RelativeChild kNumChildren = int_math::exp2(dim);
   using ChildArray = std::array<NdtreeIndex, kNumChildren>;
 
+  //! The node's resolution level in the octree
+  //! @note A height of 0 corresponds to the map’s maximum resolution.
```

Comment:
Maybe use root and leaf lingo in addition to this in here as maximum resolution still might be confusing, when root and leaf for nodes is very specific and has no dual integration ability.

Reply:
How about this explanation?
```
  //! @note A height of 0 corresponds to the map’s maximum resolution. In a 
  //!       fully allocated tree, all leaf nodes are at height 0. Increasing 
  //!       the height by 1 doubles the node size along each dimension. The 
  //!       root node corresponds to the map's lowest resolution, and its height 
  //!       matches the configured tree height.
```